test/headless/src/page/profile.ts

```diff
@@ -1,4 +1,5 @@
 //profile page
-// import * as puppeteer from 'puppeteer'
 
-// export const selectors: {[key: string]: string} = {}
+export const selectors: {[key: string]: string} = {
```

Comment:
Instead of `{[key: string]: string}`, you may want to use the builtin `Record` type in the future, e.g.

```ts
Record<string, string>
```

It's basically a shortcut for the other.

Reply:
Thanks, I'll add going forward